rsearch/config.ini

```diff
@@ -0,0 +1,18 @@
+; comment
+[api]
+url=http://192.168.0.10:8080
+namespaceUrl=api/v1/namespaces/?watch=true
+
+[resource]
+;type=builtin # TBD
+type=3rdParty
+;namespaced=false # TBD
```

Comment:
Could we add some comments after or before each config item to explain what the config is for? For example, it's not clear what "namespaced" means.